src/components/AddPlaidBankAccount.js

```diff
@@ -146,17 +161,20 @@ class AddPlaidBankAccount extends React.Component {
             value: index, label: `${account.addressName} ${account.accountNumber}`,
         }));
         const {icon, iconSize} = getBankIcon(this.state.institution.name);
+        const plaidLinkToken = !_.isEmpty(this.props.plaidLinkToken) || !_.isEmpty(this.props.existingPlaidLinkToken);
```

Comment:
NAB : I'd call this something like `hasPlaidLinkToken` since it's a bool.

Reply:
And then the `plaidLinkTokenToUse` can become `plaidLinkToken`.

Reply:
+1

Reply:
I agree, that sounds better